### ISSUE TITLE
[HOLD] Manually CPing a pull request does not add the PR to the checklist

### ISSUE THREAD

**Issue by roryabraham** [Expensify/App#4379]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Lock the deploy checklist
1. Merge a PR.
1. Manually cherry-pick the PR.

## Expected Result:
The deploy checklist should be updated with the new PR.

## Actual Result:
The deploy checklist is not updated.

## Workaround:
N/A

## Platform:
GitHub only

**Version Number:** `1.0.82-2`

[View all open jobs on Upwork](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2)

**Comment by roryabraham** [comment]
No update here, but it would be a great FP for someone looking to familiarize themselves with the NewDot deploy code.

**Comment by roryabraham** [comment]
Sorry, no update again.

**Comment by roryabraham** [comment]
PR is up

**Comment by botify** [comment]
🚀 [Deployed](https://github.com/Expensify/App/actions/runs/1183075988) to production by @roryabraham in version: 1.0.88-2 🚀

 platform | result 
 ---|--- 
🤖 android 🤖|success ✅ 
🖥 desktop 🖥|success ✅
🍎 iOS 🍎|failure ❌ 
🕸 web 🕸|success ✅

**Comment by roryabraham** [comment]
I had to revert the PR to fix this, so I'm reopening it.

**Comment by roryabraham** [comment]
Just realized that, while I fixed the bigger related bug, I never actually came back to fix this one. Should be a pretty quick-and-easy fix.

**Comment by roryabraham** [comment]
New PR is up

**Comment by roryabraham** [comment]
Had to revert the solution again 😞

**Comment by roryabraham** [comment]
Going to work on figuring out https://github.com/Expensify/App/issues/4977 first

**Comment by roryabraham** [comment]
Some additional context in [this slack thread](https://expensify.slack.com/archives/C07J32337/p1640115752259200) but the solution for this issue would likely also solve https://github.com/Expensify/App/issues/6863.

*TL;DR* we want to make sure that the CP PR is merged and the tag for the new version is pushed before updating the checklist.

**Comment by roryabraham** [comment]
Hopefully got a solution up for https://github.com/Expensify/App/issues/4977, so this PR is next.

**Comment by roryabraham** [comment]
Also related: https://github.com/Expensify/App/issues/7178

**Comment by roryabraham** [comment]
Note to self, the comments from https://github.com/Expensify/App/pull/7802 also won't work with a manual CP

**Comment by roryabraham** [comment]
No update on deploy stuff rn

**Comment by roryabraham** [comment]
on HOLD

**Comment by Beamanator** [comment]
FYI in this checklist, it looks like OSBotify did add 2 `[CP Staging]` PRs to the checklist, but we had more than 2 CP'd PRs yesterday 😅 

PRs that got added to checklist:
- https://github.com/Expensify/App/pull/23988
- https://github.com/Expensify/App/pull/24003

PRs that got CP'd yesterday:
- https://github.com/Expensify/App/pull/23988
- https://github.com/Expensify/App/pull/24003
- https://github.com/Expensify/App/pull/24025
- https://github.com/Expensify/App/pull/24010

**Comment by roryabraham** [comment]
Actually going to close this as a dupe of https://github.com/Expensify/App/issues/21763 because that has better bread crumbs at this point
